sequence_task_3/student/include/impl.h

```diff
@@ -11,4 +12,19 @@
  *
  * @warning YOU MUST USE STRING STREAMS AND STREAM ITERATORS!
  */
-std::string reverseWords(const std::string& input);
+std::string reverseWords(const std::string& input)
+{
+    std::istringstream iss(input);
+    std::vector<std::string> words(std::istream_iterator<std::string>{iss}, std::istream_iterator<std::string>());
+    std::string result = "";
+
+    std::reverse(words.begin(),words.end());
```

Comment:
You can avoid std::reverse call here, if you'll use reverse iterators or front inserter.

Reply:
done